docs/self-managed/operational-guides/update-guide/introduction.md

```diff
@@ -23,6 +23,10 @@ Update from 8.3.x to 8.4.0
 <!-- [Release notes](https://github.com/camunda/camunda-platform/releases/tag/8.4.0) -->
 <!-- [Release blog](ADD RELEASE BLOG LINK) -->
 
+:::note
```

Comment:
I personally would not add this note here. We already have the notes on the individual update guides, so I think it is redundant and just creates extra effort.

WDYT?